homeassistant/components/hyperion/__init__.py

```diff
@@ -96,6 +118,30 @@ async def _create_reauth_flow(
     )
 
 
+def listen_for_instance_updates(
```

Comment:
If this is called from an async context, decorate it with `@callback`:
```suggestion
@callback
def listen_for_instance_updates(
```